bundle/generate/downloader.go

```diff
@@ -161,7 +168,7 @@ func (n *Downloader) FlushToDisk(ctx context.Context, force bool) error {
 			return err
 		}
 		errs.Go(func() error {
-			reader, err := n.w.Workspace.Download(errCtx, filePath)
+			reader, err := n.w.Workspace.Download(errCtx, filePath, workspace.DownloadFormat(workspace.ExportFormatSource))
```

Comment:
We can default to the server-side export format that is persisted for the notebook. We already fetch this in the workspace filer:
1. https://github.com/databricks/cli/blob/79f2ddad91e7193d59194a01f5cfdf27582a5581/libs/filer/workspace_files_client.go#L51
2. https://github.com/databricks/cli/blob/79f2ddad91e7193d59194a01f5cfdf27582a5581/libs/filer/workspace_files_client.go#L347

We should not default to python source files because they are lossy. You lose out on the notebook metadata like cell output.

Reply:
Yes, good point, `return_export_info ` is what I was looking for, thanks!

Reply:
Fixed here: https://github.com/databricks/cli/pull/3316/commits/64e10de096e2a9b6723f59e2263dcd4e24a0c27f